src/dump.rs

```diff
@@ -207,15 +208,33 @@ impl DumpClassMember {
             },
         }
     }
+
+    pub fn add_tag(&mut self, tag: &str) {
+        let mut tags = match self {
+            DumpClassMember::Property(inner) => {
+                &mut inner.tags
+            },
+            DumpClassMember::Function(inner) => {
+                &mut inner.tags
+            },
+            DumpClassMember::Event(inner) => {
+                &mut inner.tags
+            },
+            DumpClassMember::Callback(inner) => {
+                &mut inner.tags
+            },
+        };
+        tags.insert(tag.to_owned());
+    }
 }
 
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(rename_all = "PascalCase")]
 pub struct DumpClassProperty {
     pub name: String,
 
-    #[serde(default = "Vec::new")]
-    pub tags: Vec<String>,
+    #[serde(default = "BTreeSet::new")]

```

Comment:
```suggestion
    #[serde(default)]
```

Same default change here.